src/core/connect-extension/conn-ext-build-state/build-preferences.ts

```diff
@@ -14,6 +15,14 @@ export const buildPreferences = (trimmedPreferences: IExtStorage.IStoragePrefere
         });
     });
 
+    // add inactive blockchains
+    BLOCKCHAIN_LIST.filter(b => !trimmedPreferences.blockchains.includes(b)).map(blockchain => {
+        blockchains[blockchain] = {
+            order: 999,
```

Comment:
Should we use 
```
 BLOCKCHAIN_LIST.filter(b => !trimmedPreferences.blockchains.includes(b)).map((blockchain, index) => {
        blockchains[blockchain] = {
            order: 999 - index,
            active: false
        };
    });
```
So that each blockchain will have an unique order? 
Or it's not necessary.

Reply:
it's not necessary, actually the order could be random, since everything in that list will be inactive :)